src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -231,10 +232,16 @@ def create_results_link(output_dir: Path, link_name: Path) -> None:
 def perform_post_processing(
     raw_output_dir: Path, final_output_dir: Path, artifact_path: Path
 ) -> None:
+    # Create RandomnessStream for post-processing
+    key = "post_processing_maps"
+    clock = lambda: pd.Timestamp("2020-04-01")
```

Comment:
This clock stresses me out. To clarify, this is a totally random date that simply does not matter, right? It has nothing at all to do w/ the sim start date?

Reply:
Correct; correct :)